sleap/gui/commands.py

```
@@ -3470,13 +3473,293 @@
+    def do_action(context: CommandContext, params: dict):
+        """
+        Extracts a video clip and saves it with pose annotations.
 
-def copy_to_clipboard(text: str):
-    """Copy a string to the system clipboard.
+        Args:
+            context: Command context, providing state and application access.
+            params: Parameters for the action, including file paths and export options.
+        """
+        from sleap.io.visuals import save_labeled_video
+        from sleap import Labels
```

Ruff (0.8.2)

`sleap.Labels` imported but unused

Remove unused import: `sleap.Labels`

(F401)